CODE_OF_CONDUCT.md

```diff
@@ -14,22 +14,22 @@ appearance, race, religion, or sexual identity and orientation.
 Examples of behavior that contributes to creating a positive environment
 include:
 
-* Using welcoming and inclusive language
-* Being respectful of differing viewpoints and experiences
-* Gracefully accepting constructive criticism
-* Focusing on what is best for the community
-* Showing empathy towards other community members
+- Using welcoming and inclusive language
+- Being respectful of differing viewpoints and experiences
+- Gracefully accepting constructive criticism
+- Focusing on what is best for the community
+- Showing empathy towards other community members
```

Comment:
Prettier it too opinionated

Reply:
I don't mind that. 😄 Looks like you mind that a lot.

Reply:
I do not really. However, it takes advantages from me. GitHub developers given it to me, and I am happy to use hyphens for the unordered lists but other developers from prettier decided to forbid it.

I like underscores for italic and bold but Prettier guys decided that this is not good too.